packages/diagrams/backend/sirius-components-diagrams/src/main/java/org/eclipse/sirius/components/diagrams/LabelStyle.java

```diff
@@ -135,6 +175,31 @@ public Builder iconURL(List<String> iconURL) {
             return this;
         }
 
+        public Builder background(String background) {
+            this.background = Objects.requireNonNull(background);
+            return this;
+        }
+
+        public Builder borderColor(String borderColor) {
+            this.borderColor = Objects.requireNonNull(borderColor);
+            return this;
+        }
+
+        public Builder borderSize(int borderSize) {
```

Comment:
Does this not require NotNull on purpose ?

Reply:
Yes, it's because these are primitive `int` type, so they cannot be null